pkgs/development/tools/misc/sqitch/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, writeScript, makeWrapper, sqitchModule }:
+stdenv.mkDerivation rec {
+  name = "sqitch-0.999";
+  buildInputs = [ makeWrapper ];
+  propagatedNativeBuildInputs = [ sqitchModule ];
+  builder = writeScript (name + "-builder.sh") ''
```

Comment:
Can you avoid using the builder here? I think this is enough:
```
unpackPhase = "true";
buildPhase = "true";
installPhase = those symlinks;
postFixup = the wrapper;
```

Also in the wrapper you shouldn't need to do that for by yourself, just `--prefix PERL5LIB $PERL5LIB`. And sqitchModule doesn't need to be propagated I think.